Library/Formula/xsd.rb

```diff
@@ -0,0 +1,17 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
+  sha256 "56571e446ae49935611cbf46484a50c1a0afd3bb7987be728b8dcdd3de34b897"
+
+  depends_on 'xerces-c'
+
+  def install
+    bin.install "bin/xsd"
+    include.install "libxsd/xsd"
+  end
+
+  test do
+    system "#{bin}/xsd version"
```

Comment:
Could you add a test that does something more substantial than e.g. `--version` or `--help`? Thanks!

Also, use `system "foo", "bar"` instead of `system "foo bar"`.